prediction/PedestrianCollisionEvent[] collision_event_list
